hknweb/models.py

```diff
@@ -28,3 +32,20 @@ def clean(self):
         if self.phone_number:
             self.phone_number = re.sub("[^0-9]", "",self.phone_number)
             self.phone_number = "("+self.phone_number[0:3]+") "+self.phone_number[3:6]+"-"+self.phone_number[6:]
+
+class Announcement(models.Model):
+    """
+    Model for an announcement. Created by VP or some other superuser.
+    Displayed on the candidate portal. The title will be displayed in bold,
```

Comment:
This docstring seems a bit inaccurate...

Reply:
Ok fixed comment